target-typescript/stringmap.d.ts

```diff
@@ -11,3 +11,16 @@ export declare class StringMap<V, K extends string = string> {
     keys(): K[];
     entries(): [K, V][];
 }
+export declare class NumberMap<V, K extends number = number> {
```

Comment:
`number` can't be usefully extended as far as I know, so I'd remove the generic from this type and just make `K` number.